### PR TITLE
Feature/database data

### PR DESCRIPTION
**Pull request by jaalazard**
## Description

Please include a summary of the change or which issue is fixed.

## Type of change

- [x ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Area of change

- [ ] Frontend
- [ x] Backend
- [ ] Other: .....

## General checklist:

- [ x] My code follows the style guidelines of this project
- [x ] I ran `bun run format`/`bun run lint` before commit
- [x ] I have performed a self-review of my own code
- [x ] I have commented my code, particularly in hard-to-understand areas
- [x ] I have made corresponding changes to the documentation where needed
- [x ] My changes generate no new warnings
- [x ] I have checked my code and corrected any misspellings
- [x ] I have updated `develop` and merged to my branch before submitting pull request
- [x ] My pull request generate no conflicts with `develop` branch
- [x ] I requested code review from other team members

## Frontend checklist:

- [ ] I followed guidelines for `HTML`/`JSX`/`Typescript`
- [ ] My `Javascript` generate no new console errors
- [ ] I tested my code cross browsers
- [ ] My slice is pixel perfect for both desktop and mobile according to design
- [ ] I conducted basic QA to assure all features are working
- [ ] I tested responsive for mobile and tablet resolutions

## Backend checklist:

- [ ] I tested admin by manually adding content from zero
- [ ] I created easy to use admin experience which is self-explanatory
- [ ] I added description to admin fields in hard-to-understand areas
- [ ] I followed guidelines for naming `Typescript` variables
- [ ] I conducted basic QA to assure all features are working

### CODE REVIEW

File: packages/backend/forge/README.md
Comment (Cardamced) [inline packages/backend/forge/README.md:1]: Vraiment besoin de push le readme du dossier forge ? C'est juste une question... ;-)

**Review by Cardamced — DISMISSED** [review]
Moyennant le commentaire que j'ai laissé, c'est bon pour moi.

**Review by No3ml3 — DISMISSED** [review]

**Review by No3ml3 — DISMISSED** [review]

**Review by Cardamced — DISMISSED** [review]
Ok pour moi.

**Review by No3ml3 — DISMISSED** [review]

**Review by Cardamced — DISMISSED** [review]
Cool.

File: packages/backend/migrations/src/migrations/20231122143414-database-structure.ts [hunk 1]
```diff
@@ -8,7 +8,7 @@ export async function up(db: Kysely<Database>): Promise<void> {
       CREATE TABLE IF NOT EXISTS glass (
         id INT UNSIGNED AUTO_INCREMENT PRIMARY KEY,
         name VARCHAR(50) NOT NULL,
-        capacity TINYINT UNSIGNED NOT NULL,
+        capacity SMALLINT UNSIGNED NOT NULL,
```
Comment (YDrogen) [inline packages/backend/migrations/src/migrations/20231122143414-database-structure.ts]: Non, la migration a déjà été merge, il faut en faire une nouvelle

**Review by YDrogen — CHANGES_REQUESTED** [review]

**Review by No3ml3 — DISMISSED** [review]

**Review by Cardamced — DISMISSED** [review]
Bien.

File: packages/backend/migrations/src/migrations/20231201111652-new-database-structure.ts
Comment (YDrogen) [inline packages/backend/migrations/src/migrations/20231201111652-new-database-structure.ts:1]: Toutes ces tables existent déjà, non ? Pourquoi les recréer ?

**Review by YDrogen — CHANGES_REQUESTED** [review]

**Review by No3ml3 — DISMISSED** [review]

**Review by Cardamced — DISMISSED** [review]
Parfait.

File: packages/backend/migrations/src/migrations/20231201111652-edit-glass-table.ts
Comment (YDrogen) [inline packages/backend/migrations/src/migrations/20231201111652-edit-glass-table.ts:1]: Maintenant, il manque le `down`
Chaque migration doit avoir un `up` et un `down`

**Review by YDrogen — CHANGES_REQUESTED** [review]

**Review by No3ml3 — DISMISSED** [review]

**Review by Cardamced — DISMISSED** [review]
Ok pour moi.

File: packages/backend/migrations/src/migrations/20231201111652-edit-glass-table.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+      MODIFY capacity SMALLINT UNSIGNED NOT NULL;
+    `.execute(trx);
+  });
+}
+
+export async function down(db: Kysely<Database>): Promise<void> {
+  await db.transaction().execute(async (trx) => {
+    await sql`
+      ALTER TABLE glass
+      MODIFY capacity TINYINT UNSIGNED NOT NULL;;
```
Comment (YDrogen) [inline packages/backend/migrations/src/migrations/20231201111652-edit-glass-table.ts]:
```suggestion
      MODIFY capacity TINYINT UNSIGNED NOT NULL;
```
:(

**Review by YDrogen — CHANGES_REQUESTED** [review]

**Review by No3ml3 — APPROVED** [review]

**Review by Cardamced — APPROVED** [review]

**Review by YDrogen — APPROVED** [review]

**Pull request merged**